internal/templates/03-assert.yaml.tmpl

```diff
@@ -15,7 +15,9 @@ commands:
 - command: ${KUBECTL} wait {{ $resource.KindGroup }}/{{ $resource.Name }} --for=delete --timeout 10s
 {{- end }}
 {{- end }}
+{{- if not .TestCase.OnlyCleanUptestResources }}
 - command: ${KUBECTL} wait managed --all --for=delete --timeout 10s
+{{- end }}
```

Comment:
So, we are actually deleting the provisioned resources elsewhere and this waiting condition is the breaking part in case there are other managed resources in the cluster that are actually not provisioned by the uptest run. Maybe, we can also consider getting rid of this command all together.

Reply:
@ulucinar Thank you for highlighting this. We have such a check because of the assumption you mentioned. I wanted to keep this change small by being more conservative here. It seems reasonable to remove it in the next iterations.